packages/cspell-types/src/CSpellSettingsDef.ts

```diff
@@ -156,7 +199,11 @@ export interface Settings extends ReportingConfiguration, BaseSetting, PnPSettin
      */
     enableFiletypes?: LanguageIdSingle[];
 
-    /** Additional settings for individual languages. */
+    /**
+     * Additional settings for individual languages.
+     *
+     * See the [language settings documentation](https://cspell.org/docs/configuration/language-settings/) for more details.
```

Comment:
The current link is: `[Language Settings](https://cspell.org/configuration/language-settings/)`

Reply:
can you offer this as a suggestion so i can click the button "commit suggestion"?